jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java

```diff
@@ -175,11 +177,24 @@ protected void addProcessIdFilter() {
                                     f -> addSearchFilter(f, equalsTo(COLUMN_PROCESS_ID, f.getValue())));
     }
 
+    protected void addSLAComplianceFilter() {
+        Map<String, String> aliasMap = new HashMap<String, String>();
+        aliasMap.put("0", constants.SlaNA());
+        aliasMap.put("1", constants.SlaPending());
+        aliasMap.put("2", constants.SlaMet());
```

Comment:
Probably better to use the static references we already have:
```suggestion
        aliasMap.put(String.valueOf(ProcessInstance.SLA_MET), constants.SlaMet());
```